base/linalg/dense.jl

```diff
@@ -5,6 +5,7 @@
 ## BLAS cutoff threshold constants
 
 const SCAL_CUTOFF = 2048
+const TRIA_CUTOFF = 2048
```

Comment:
Maybe it is better to use `eltype`, and then assume a `sizeof` of ... 16 for abstract types? Or just pick the triangular one? I guess it's fair that matrices that aren't that strict about their types can't expect an optimisation like this.